starter-code/routes/index.js

```diff
@@ -1,9 +1,76 @@
-const express = require('express');
-const router  = express.Router();
+const express = require("express");
+const router = express.Router();
+const userModel = require("./../models/user");
+const bcrypt = require("bcrypt");
+// const createUser = require("./../views/create-user");
 
 /* GET home page */
-router.get('/', (req, res, next) => {
-  res.render('index');
+router.get("/", (req, res, next) => {
+  res.render("index");
+});
+
+router.get("/user", (req, res) => {
+  res.render("user");
+});
+
+//signin
+
+router.post("/create-user", (req, res, next) => {
+  const user = req.body; // req.body contains the submited informations (out of post request)
+
+  if (!user.username || !user.password) {
+    res.render("index", { message: "Fill all the fields" });
+  } else {
+    userModel
+      .findOne({ username: user.username })
+      .then(dbRes => {
+        if (dbRes) {
+          res.redirect("/user");
```

Comment:
normally here because you are creating a user, if you get a response from the DB it means that a user already exists with this username so the new user should find another name. He shouldn't be redirect to the private page